examples/gridtools/simple/example-01-laplacian.cpp

```diff
@@ -96,19 +97,20 @@ void write() {
   //
   // Allocate the 2D arrays phi and lap and fill it with some random numbers
   //
-  meta_data_t meta_data(N, M, 1);
+  storage_info_t storage_info(N, M, 1);
 
-  storage_t phi(meta_data, "phi", -1);
-  storage_t lap(meta_data, "lap", -1);
+  storage_t phi(storage_info, -1., "phi");
+  storage_t lap(storage_info, -1., "lap");
 
   std::default_random_engine gen;
   std::uniform_real_distribution<double> dist(0.0, 1.0);
+  auto phi_view = make_host_view(phi);
   for(int i = 0; i < N; ++i)
     for(int j = 0; j < M; ++j)
-      phi(i, j, 0) = dist(gen);
-  
+      phi_view(i, j, 0) = dist(gen);

```

Comment:
you could also use the lambda init ctor `storage_t phi(storage_info, [&](int i, int j, int k) { return dist(gen); }, "phi");`. but no need to change this.